schema/private.openapi.yaml

```diff
@@ -165,6 +165,8 @@ components:
           $ref: '#/components/schemas/RunInputHosts'
         recipient_config:
           $ref: '#/components/schemas/RecipientConfig'
+        account:
```

Comment:
I believe I added this here so that if the account field was passed it could be captured and used for logging in the v2 endpoint. It should be removed now

Reply:
I may have confused you by what I said earlier. What I meant was that if we happen to have account (e.g. it comes in the identity header anyway, alongside org_id) then why not log it. However, we should not be going as far as changing APIs to re-introduce a deprecated field, though.